src/peer/negotiator.js

```diff
@@ -114,6 +114,23 @@ class Negotiator extends EventEmitter {
         }
       });
 
+      if (this._pc.getSenders().every(sender => {
+        return sender.track.kind !== 'audio';
```

Comment:
Personally I prefer extracting this condition to another method like,

## Before

```
      if (this._pc.getSenders().every(sender => {
        return sender.track.kind !== 'audio';
      }))
```

## After

```
      if (_onlyVideoTracksExist(this._pc)) { ...}
```

This explains what we're doing in the condition.